crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -35,25 +27,24 @@ use tracing::{
     instrument,
 };
 
+const EXIT_CONFIG_CODE: i32 = 78;
+
 #[tokio::main]
 async fn main() -> Result<()> {
-    run().await?;
+    if let Err(e) = run().await {
+        eprintln!("Exited with error: {e}");
+        // FIXME: might have to bubble up exit codes, since we might need 
+        //        to exit with other exit codes if something else fails
+        std::process::exit(EXIT_CONFIG_CODE);
+    };
     Ok(())
 }
 
 #[instrument(name = "astria_conductor::run")]
 async fn run() -> Result<()> {
-    let args = Cli::parse();
-    // hierarchical config. cli args override Envars which override toml config values
-    let conf: Config = Figment::new()
-        .merge(Toml::file("ConductorConfig.toml"))
-        .merge(Env::prefixed("RUST_").split("_").only(&["log"]))
-        .merge(Env::prefixed("ASTRIA_"))
-        .merge(Serialized::defaults(args))
-        .extract()?;
-
-    telemetry::init(std::io::stdout, conf.log.as_deref().unwrap_or("info"))
-        .expect("failed to initialize telemetry");
+    let conf = config::get().wrap_err("failed to read config")?;
+
+    telemetry::init(std::io::stdout, &conf.log).wrap_err("failed to initialize telemetry")?;
 
     info!("Using chain ID {}", conf.chain_id);
```

Comment:
While we are changing configs anyway, can you serialize the config as json and print all of it here instead of just a few bits over several lines? Have a look at how the other services are doing it. It's good practice to tell people immediately what the config is at service startup.